app/views/support_interface/zendesk/index.html.erb

```diff
@@ -81,6 +81,7 @@
     <span class="govuk-caption-m">
       Showing <%= @zendesk_delete_requests.size %> of <%= @total %> total
     </span>
+    <%= tag.a "Export to CSV", class: 'govuk-link', href: support_interface_zendesk_export_path(format: :csv) %>
```

Comment:
Does `govuk_link` work?

Reply:
Yes it does. Below is what the link looks like

<img width="777" alt="Screenshot 2022-09-07 at 11 43 29" src="https://user-images.githubusercontent.com/547202/188859409-ac4c9148-d24f-4694-8608-fe15e73960a2.png">

Reply:
```suggestion
    <p>
      <%= govuk_link_to "Export to CSV", support_interface_zendesk_export_path(format: :csv) %>
    </p>
```

Reply:
Also this shouldn't in in the `h2`; could it be in a separate `<p>`?